paddle/fluid/operators/dropout_op_test.cc

```diff
@@ -83,22 +88,22 @@ void Compare(f::Scope* scope, const p::DeviceContext& ctx) {
   }
 }
 
-// TODO(wyi): Due to
-// https://github.com/PaddlePaddle/Paddle/issues/9507, I temporarily
-// disable this test to remove the prevention of the merge of
-// unrelated PRs.
-/*
 TEST(Dropout, CPUDense) {
   f::Scope scope;
   p::CPUPlace place;
   p::CPUDeviceContext ctx(place);
-  Compare(scope, ctx);
+  Compare(&scope, ctx);
 }
 
+// TODO(wyi, dzhwinter): Due to
+// https://github.com/PaddlePaddle/Paddle/issues/9507, I temporarily
+// disable this test to remove the prevention of the merge of
+// unrelated PRs.
+/*
 TEST(Dropout, GPUDense) {
```

Comment:
Can you do a followup PR to enable this as well? Perhaps the Wait() you add solves the random behavior.

Reply:
Of course.